.github/workflows/inspektor-gadget.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       id: publish-registry
       uses: elgohr/Publish-Docker-Github-Action@2.8
       with:
-        name: docker.io/kinvolk/gadget
+        name: ${{ secrets.DOCKER_REPO }}
```

Comment:
Line 41, near `make kubectl-gadget`:
```
      env:
        CONTAINER_REPO: ${{ secrets.DOCKER_REPO }}
```

Maybe the same for the other `make` invokations.

Reply:
Done.